python/cac_tripplanner/destinations/management/commands/load_events.py

```diff
@@ -0,0 +1,93 @@
+from datetime import datetime
+import urllib2
+from xml.dom import minidom
+
+from django.core.management.base import BaseCommand
+from django.contrib.gis.geos import Point
+from destinations.models import FeedEvent
+
+from pytz import timezone
+
+
+class Command(BaseCommand):
+    args = ''
+    help = 'Load the current Uwishunu feed into the FeedEvent table'
+
+    def property_exists(self, item, property_name):
+        """ Check if a property exists as a child of the given element """
+        return item.getElementsByTagName(property_name).length > 0
+
+    def get_property(self, item, property_name):
+        """ Retrieve the value of an xml property name from another xml element """
+        elements = item.getElementsByTagName(property_name)
+        if elements.length == 0:
+            return None
+        return elements[0].firstChild.data
+
+    def handle(self, *args, **options):
+        """ Retrieve and populate FeedEvent table from an RSS Feed """
+
+        url = 'http://www.uwishunu.com/category/events/feed/'
+        # Get 403 forbidden without changing user-agent
+        headers = {'User-Agent': 'Mozilla/5.0 (X11; Linux i686; rv:10.0) Gecko/20100101 Firefox/10.0'}
+        request = urllib2.Request(url, None, headers)
```

Comment:
It'd be good to have some error handling here, with problems being written to a log file in order to make it easy to diagnose failures.